### PR TITLE
Translate doc first steps to Chinese

### PR DESCRIPTION
**Pull request by waynerv**
@Dustyposa @RunningIkkyu Please take a look.

### CODE REVIEW

**Review by yanever — APPROVED** [review]

File: docs/zh/docs/tutorial/first-steps.md [hunk 1]
```diff
@@ -0,0 +1,335 @@
...
+
+#### API "模式"
+
+在这种场景下，OpenAPI 是一种规定如何定义 API 模式的规范。
+
+定义的 OpenAPI 模式将包括你的 API 路径，以及它们可能使用的参数等等。
+
+#### 数据 "模式"
+
+"模式"这个术语也可能指的是某些数据比如 JSON 的结构。
```
Comment (RunningIkkyu) [inline docs/zh/docs/tutorial/first-steps.md:86]: Maybe change `""` or `”“` to `「」`

**Review by RunningIkkyu — APPROVED** [review]
Maybe we can change all quote marks into `「」`.

File: docs/zh/docs/tutorial/first-steps.md (hunk 1 above)
Comment (waynerv) [reply docs/zh/docs/tutorial/first-steps.md]: Good idea, already done.

**Review by Laineyzhang55 — APPROVED** [review]
LGTM

**Comment by tiangolo** [comment]
Awesome, thanks for your contribution @waynerv ! :rocket: :tada: 

And thanks for the review/approvals @RunningIkkyu , @Laineyzhang55 , @yanever :bow: :coffee:

**Pull request merged**